src/server/services/toeristische-verhuur.ts

```diff
@@ -155,7 +155,7 @@ export function transformVergunningenToVerhuur(
             : 0,
       };
     })
-    .sort(dateSort('dateStart', 'asc'));
+    .sort(dateSort('dateStart', 'desc'));
```

Comment:
This is not correct. We want to display the dates of geplande verhuur that start the soonest first. So we should split up the sort logic by constructing 2 arrays, sort them accordingly (gepland => dateStart.asc), (geannuleerd + afgelopen => dateStart.desc), (vergunning => dateStart.desc).